src/templates/api_reference/resource.ejs

```diff
@@ -0,0 +1,70 @@
+<%
+var thisType = plural(cap(resource.name));
+
+%>---
+parent-title: API Reference
+title: <%= thisType %>
+description:
+_template: dev-api-reference
+_layout: dev
+
+
+# Sidebar nav
+side_nav:
+  <% _.forEach(resource.action_classes, function(action_class) {%><%= '-'%>
+    <%= 'name: ' + action_class.name%>
```

Comment:
Constant string values in templates are often best expressed as raw values in the template itself. So this might be cleaner as:

    name: <%= action_class.name %>

etc.